tests/Kron.py

```diff
@@ -259,7 +356,7 @@ def measureKronInPython(self, objImg, xcen, ycen, nsigma, kfac, nIterForRadius,
 
         return R_K, sumI, 0, False
 
-    def testEllipticalGaussian(self):
+    def xtestEllipticalGaussian(self):
```

Comment:
Just a failure to look closely before I commited.  This test takes a long time, and I was "temporarily" commenting it out.  Sloppy me.